### PR TITLE
feat: added getInflationReward method

### LINKED ISSUE #316: No method for getInflationReward
This JSON RPC method...

https://docs.solana.com/developing/clients/jsonrpc-api#getinflationreward

...is missing from the client.

### PR DESCRIPTION
**Pull request by kadern0**
Fixes #316 

Signed-off-by: Pablo Caderno <kaderno@gmail.com>

### CODE REVIEW

**Comment by kevinheavey** [comment]
Can you add tests to `tests/integration/test_async_http_client.py` and `tests/integration/test_http_client.py` please? Imitating an existing test should suffice

File: src/solana/rpc/async_api.py [hunk 1]
```diff
@@ -574,7 +593,7 @@ async def get_multiple_accounts(
             >>> from solana.publickey import PublicKey
             >>> solana_client = AsyncClient("http://localhost:8899")
             >>> pubkeys = [PublicKey("6ZWcsUiWJ63awprYmbZgBQSreqYZ4s6opowP4b7boUdh"), PublicKey("HkcE9sqQAnjJtECiFsqGMNmUho3ptXkapUPAqgZQbBSY")]
-            >>> (await solana_client.get_multiple_accounts(pubkeys)).value[0].lamports # doctest: +SKIP
+            >>> (await solana_client.multiple_accounts(pubkeys)).value[0].lamports # doctest: +SKIP
```
Comment (kevinheavey) [inline src/solana/rpc/async_api.py:596]: Guessing this is accidental, please undo

Comment (kadern0) [reply src/solana/rpc/async_api.py:596]: :thinking: I don't have that on my local code. I've probably messed up with the commits. That was a mistake.

**Comment by kadern0** [comment]
@kevinheavey, I've added the test code but I'm unable to make it pass. Could you have a look?

**Comment by kevinheavey** [comment]
@kadern0 maybe try it with Processed commitment?

**Comment by kadern0** [comment]
@kevinheavey, using `Processed` I get:
```
resp = InvalidParamsMessage {
    message: "Method does not support commitment below `confirmed`",
}

```
And with `Confirmed` or `Finalized` I get:
```
raw = '{"jsonrpc":"2.0","error":{"code":-32004,"message":"Block not available for slot 8192"},"id":0}\n'
parser = <class 'solders.rpc.responses.GetInflationRewardResp'>
```

**Pull request closed without merging**